tests/test_amdgpu.cpp

```diff
@@ -56,11 +59,12 @@ static void test_amdgpu_get_instant_metrics(void **state) {
     amdgpu_get_instant_metrics(&metrics);
     assert_int_equal(metrics.gpu_load_percent, 100);
     assert_float_equal(metrics.average_gfx_power_w, 6.161, 0);
+    assert_float_equal(metrics.average_cpu_power_w, 9.235, 0);
     assert_int_equal(metrics.current_gfxclk_mhz, 1040);
     assert_int_equal(metrics.current_uclk_mhz, 687);
     assert_int_equal(metrics.gpu_temp_c, 81);
     assert_int_equal(metrics.soc_temp_c, 71);
-    assert_int_equal(metrics.apu_cpu_temp_c, 655);
+    assert_int_equal(metrics.apu_cpu_temp_c, 80);
```

Comment:
Maximum of:
- 80 (`0x1f72`)
- 74 (`0x1d01`)
- 77 (`0x1e2d`)
- 73 (`0x1e2d`)

from [gpu_metrics_apu](https://github.com/flightlessmango/MangoHud/blob/0c155b87b01d88eb8c6d03939b615c5fed664fd2/tests/gpu_metrics_apu) at byte 0x08